app/src/main/java/com/android/voyageur/ui/notifications/AndroidNotificationProvider.kt

```diff
@@ -0,0 +1,29 @@
+package com.android.voyageur.ui.notifications
+
+import android.content.Context
+import androidx.core.app.NotificationCompat
+import com.android.voyageur.R
+
+open class AndroidNotificationProvider(private val context: Context) : NotificationProvider {
```

Comment:
Thank you for implementing the AndroidNotificationProvider! It would be great to add some documentation to make the code easier to understand.